cluster-classification/script.whizzml

```diff
@@ -0,0 +1,56 @@
+;; Given a cluster, creates a batch centroid and uses the output dataset
+;; where each instance is labeled by centroid to create an ensemble (random forest), 
+;; and returns the importance of each field averaged over the ensemble.
+
+;; Given a cluster, returns the dataset that created that cluster appended with the cluster
+;; that instance came from.
+(define (label-by-cluster cl-id)
+  (let (cluster-ds (get-in (fetch cl-id) ["dataset"])
+        cluster-fields (get-in (fetch cl-id) ["input_fields"])
+        batchcentroid-id (create-and-wait-batchcentroid {"cluster" cl-id
+                                                         "dataset" cluster-ds
+                                                         "output_dataset" true
+                                                         "output_fields" cluster-fields}))
+    (get-in (fetch batchcentroid-id) ["output_dataset_resource"])))
```

Comment:
Since you're only using one key with the `get-in`, a regular `get` would be better.